modules/govuk_jenkins/templates/jobs/deploy_emergency_banner.yaml.erb

```diff
@@ -25,7 +25,59 @@
           LINK=$(escapeRakeArg $LINK)
           LINK_TEXT=$(escapeRakeArg $LINK_TEXT)
 
+          echo "Deploying emergency banner"
           ssh deploy@$(govuk_node_list -c frontend --single-node) "cd /var/apps/static && govuk_setenv static bundle exec rake \"emergency_banner:deploy[$CAMPAIGN_CLASS,$HEADING,$SHORT_DESCRIPTION,$LINK,$LINK_TEXT]\""
+
+          echo "Clearing template cache.."
+          for node in `govuk_node_list frontend`; do
+            echo "-- $node"
+            ssh deploy@$node "sudo rm /var/apps/static/public/templates/{wrapper,homepage,header_footer_only,core_layout}.html.erb"
+            ssh deploy@$node "sudo rm -rf /var/apps/{,government-}frontend/tmp/cache/*"
+          done
```

Comment:
nitpick - but think you missed a new line

Reply:
That was kinda deliberate as both steps are part of clearing the template cache.  Maybe I can clarify that with comments (or better, functions) instead.